pkgs/top-level/aliases.nix

```diff
@@ -924,6 +925,7 @@ mapAliases ({
   pgadmin3 = throw "pgadmin3 was removed for being unmaintained, use pgadmin4 instead."; # Added 2022-03-30
   pgp-tools = throw "'pgp-tools' has been renamed to/replaced by 'signing-party'"; # Converted to throw 2022-02-22
   pg_tmp = throw "'pg_tmp' has been renamed to/replaced by 'ephemeralpg'"; # Converted to throw 2022-02-22
+  pixie = throw "pixie has been removed: abandoned by upstream"; # Added 2022-04-21
```

Comment:
Would be nice to amend this message with how to install it instead; possibly pointing to NUR.

Reply:
Are you willing to make a follow-up PR for that?

Reply:
I don't see pixie in nur-combined, so pointing people to NUR wouldn't seem helpful?

Reply:
I'll merge whenever this is rebased. Folloe up can be done if and when it's added to NUR or wherever :)